app/controllers/api/validations_controller.rb

```diff
@@ -1,9 +1,16 @@
 module Api
   class ValidationsController < ApiController
+    # rubocop:disable Metrics/MethodLength
```

Comment:
Of course. Over-adherence to arbitrary 'rules' seems to be a favourite hobby of many, and generally results in terrible code. I know why the rule is there, but when I choose to break it, I do so with no qualms.